client/browser/src/shared/code-hosts/github/style.scss

```
@@ -1,3 +1,25 @@
+@mixin github-dark-theme {
+    [data-color-mode='dark'] {
+        @content;
+    }
+    [data-color-mode='auto'] {
+        @media (prefers-color-scheme: dark) {
+            @content;
+        }
+    }
+}
+
+:root {
+    // Map out CSS variables to the theme-aware GitHub ones
```

```suggestion
    // Map our CSS variables to the theme-aware GitHub ones
```
Not sure if this is a typo